pkgs/applications/networking/wgnord/default.nix

```
@@ -24,6 +24,7 @@ resholve.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace wgnord \
+      --replace 'conf_dir=/var/lib/wgnord' "conf_dir=$out/share" \
       --replace '$conf_dir/countries.txt' "$out/share/countries.txt" \
       --replace '$conf_dir/countries_iso31662.txt' "$out/share/countries_iso31662.txt"
```

`	/nix/store/9c5qm297qnvwcf7j0gm01qrslbiqz8rs-gnused-4.9/bin/sed -e "s|PRIVKEY|$privkey|" -e "s|SERVER_PUBKEY|$server_pubkey|" -e "s|SERVER_IP|$server_ip|" $conf_dir/template.conf > "$out_file"`

```suggestion
      --replace '$conf_dir/countries_iso31662.txt' "$out/share/countries_iso31662.txt" \
      --replace '$conf_dir/template.conf' "$out/share/template.conf"
```

`install -Dm 644 template.conf -t $out/share/` should be added to installPhase